pytorch_sphinx_theme/layout.html

```diff
@@ -114,7 +128,21 @@
           </li>
 
           <li>
-            <a href="{{ theme_variables.external_urls['resources'] }}">Resources</a>
+            <div class="resources-dropdown">
+              <a id="resourcesDropdownButton" data-toggle="resources-dropdown">
+                Resources
+              </a>
+              <div class="resources-dropdown-menu">
+                <a class="nav-dropdown-item" href="{{ theme_variables.external_urls['resources'] }}"">
+                  <span class=dropdown-title>Developer Resources</span>
+                  <p>Find resources and get questions answered</p>
+                </a>
+                <a class="nav-dropdown-item" href="{{ theme_variables.external_urls['features'] }}">
+                  <span class=dropdown-title>About</span>
+                  <p>Learn about PyTorch’s features and capabilities</p>
+                </a>
+              </div>
+            </div>
```

Comment:
This section seems to be causing the render issues cause dropdown wasn't allowed before